calendar-bundle/contao/templates/calendar/cal_mini.html5

```diff
@@ -17,9 +17,9 @@
     <tr>
       <?php foreach ($week as $day): ?>
         <?php if (!empty($day['href'])): ?>
-          <td<?php if (!empty($day['class'])): ?> class="<?= $day['class'] ?>"<?php endif; ?>><a href="<?= $day['href'] ?>" title="<?= $day['title'] ?>"><?= $day['label'] ?></a></td>
+          <td<?php if (!empty($day['class'])): ?> class="<?= $day['class'] ?>"<?php endif; ?> headers="month"><a href="<?= $day['href'] ?>" title="<?= $day['title'] ?>"><?= $day['label'] ?></a></td>
```

Comment:
This should point to both, the `month` and the `<?= $day['name'] ?>` header I thinnk.